app/views/postponed_import_vat.scala.html

```diff
@@ -124,13 +124,24 @@
         message = "cf.account.pvat.historic.description")
         
         @link(
-            pClass = "govuk-body govuk-!-margin-bottom-9",
+            pClass = "govuk-body govuk-!-margin-bottom-3",
```

Comment:
the default spacing has been `govuk-!-margin-bottom-9` why are we changing to `govuk-!-margin-bottom-3` ?